README.md

```diff
@@ -89,12 +90,22 @@ Full list of the options
 
 Lists should be given with JSON array, do not use both wait-list and skip-list together
 
-- Each item should have a "workflowFile" field, and they can optionally have a "jobName" field.
-- If no jobName is specified, all the jobs in the workflow will be targeted.
-- wait-list:
-  - If the checkRun for the specified name is not found, this action raise errors by default.\
-    You can disable this validation with `"optional": true` or use the `startupGracePeriod` that described in following section
-  - Wait for all event types by default, you can change with `"eventName": "EVENT_NAME_AS_push"`.
+### Common spec
+
+- Must specify `workflowFile`
+- Optionally specify `jobName`\
+  If no `jobName` is specified, all the jobs in the workflow will be targeted.
+- Optionally specify `eventNames`\
+  If no `eventNames`, or empty `[]` is specified, all events will be targeted.
```

Comment:
Avoided to inherit from root event-list with the spec complexity, however it is still the reasonable behavior, I think 🤔

Reply:
Adding validation for clearly incorrect combinations of root event-lists and event names in other lists may help us.

Reply:
Actually I don't need to implement the inheritance if filtering before fetching from API https://github.com/kachick/wait-other-jobs/pull/1030#discussion_r2280992809.
It should result same.